assets/src/edit-story/components/library/panes/text/textSets/textSets.js

```diff
@@ -51,6 +51,17 @@ const CategoryWrapper = styled.div`
   margin-right: -${PANE_PADDING};
 `;
 
+const CATEGORIES = {
```

Comment:
We might wanna add context for these, not sure.

Reply:
Not sure either. All would have the same `Text set category` then. WDYT?